gradle/libs.versions.toml

```diff
@@ -9,7 +9,7 @@ maven = "3.8.6"
 mavenAnnotations = "3.6.4"
 mavenEmbedder = "3.8.6"
 mavenWagon = "3.4.3"
-graalvm = "22.3.3"
+graalvm = "22.3.5"
```

Comment:
@wirthi https://www.graalvm.org/release-notes/22_3/ does not show that we really have a 22.3.5 release but the release calendar suggest that we should have a 22.3.5 by now.